Source/Modules/php.cxx

```diff
@@ -2007,6 +2013,10 @@ class PHP : public Language {
 	  if (value) {
 	    Printf(pragma_phpinfo, "%s\n", value);
 	  }
+	} else if (Strcmp(type, "version") == 0) {
+	  if (value) {
+	    Printf(pragma_version, "%s", value);
```

Comment:
Yes, exactly.  That string is in the parse tree, so should remain valid for as long as we need it to.